### DISCUSSION TITLE
【学生は申し込め】GitHub Copilotを申請しよう – Yosuke Abe

### DISCUSSION THREAD

**Discussion by giscus** [yo5uke/yo5uke.github.io#4] [General]
# 【学生は申し込め】GitHub Copilotを申請しよう – Yosuke Abe

https://yo5uke.github.io/blog/240918_github_copilot/

**Comment by Shoya1015** [comment]
帰省中にやろうとしたら、「申請場所が学校から遠すぎる！そこにいる理由を証明しろ！」みたいに言われてめんどくさかったので、申請するなら学校に限りますね。

**Reply by yo5uke** [reply]
有益な情報助かります！そんなトラップがあったとは…。
